styles.css

```diff
@@ -67,12 +67,186 @@ h1 {
   color: #1a2236;
 }
 
-div.bottom-img {
-  background-image: url(./Images/image_geometry_2.svg);
-  background-position: top;
-  background-size: 100%;
-  margin-top: 1.5%;
-  margin-bottom: 10px;
+.bottom-img {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.bg-bottom {
+  width: 100%;
+  max-width: 100%;
+}
+
+.proj-section {
+  display: flex;
+  flex-direction: column;
+  margin-top: 20px;
+}
```

Comment:
Good job implementing the projects section :clap: , however according to the requirements _**"In order to lay out the elements in "Works" section you should use CSS Grid and Flexbox combined (it is mandatory that you use both). Use CSS Grid in the main container that contains all of the cards and use Flexbox to lay out an individual card."**_
Please use `grid` for the layout of the whole works `section` and `flex` box to for the layout of each individual project `articles`, this is because we need you to be a `flex` ninja :ninja: and `grid` master as these are very crucial skills in front-end development and will make your overall site responsive

Reply:
Thanks for your review. I have made changes as per project requirements.